cli/main.rs

```diff
@@ -526,8 +524,9 @@ async fn install_command(
   let mut preload_flags = flags.clone();
   preload_flags.inspect = None;
   preload_flags.inspect_brk = None;
```

Comment:
The `#[derive(Clone)]` on `Flags` is only still necessary because of this clone here. I wonder if there's a nice way that we could remove `#[derive(Clone)]` on `Flags` and still support this scenario.